modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/configuration/permission/propagation/DLFolderPortletConfigurationPermissionPropagation.java

```diff
@@ -62,8 +123,26 @@ protected boolean getDefaultPermissionPropagation(
 	@Override
 	protected String[] getModelResources() {
 		return new String[] {
-			DLFolder.class.getName(), DLConstants.RESOURCE_NAME
+			DLConstants.RESOURCE_NAME, DLFolderConstants.getClassName(),
+			_CHILD_DOCUMENTS_RESOURCE_NAME
 		};
 	}
 
+	private long _getClassPK(PortletRequest portletRequest) {
+		long[] resourcePrimKeys = getClassPKs(portletRequest);
+
+		return resourcePrimKeys[0];
+	}
+
+	private static final String _CHILD_DOCUMENTS_RESOURCE_NAME =
```

Comment:
_CHILD_DOCUMENT_RESOURCE_NAME or simple  _CHILD_RESOURCE_NAME

Reply:
I think _CHILD_DOCUMENT_RESOURCE_NAME would be better.

Reply:
Sorry, this tab is defined for the child documents uploaded to the folder so I think it should be `_CHILD_DOCUMENTS_RESOURCE_NAME`. I will change the tab label to `documents permissions`. Thanks.